flake.nix

```diff
@@ -6,10 +6,21 @@
     utils.url = "github:numtide/flake-utils";
   };
 
-  outputs = { self, nixpkgs, utils, }: utils.lib.eachDefaultSystem (system: let
-    pkgs = import nixpkgs { inherit system; };
-  in {
-    devShells.default = import ./Toolchain { inherit pkgs; };
-    devShells.ladybird = import ./Ladybird { inherit pkgs; };
-  });
+  outputs =
+    {
+      self,
+      nixpkgs,
+      utils,
+    }:
+    utils.lib.eachDefaultSystem (
+      system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+      in
+      {
+        devShells.default = import ./Toolchain { inherit pkgs; };
+        devShells.ladybird = import ./Ladybird { inherit pkgs; };
+        formatter = pkgs.nixfmt-rfc-style;
```

Comment:
Can you put formatting and manual changes to this file in separate commits? Reformat commits should do just the reformatting.

Reply:
👍🏾

Reply:
Thanks! Commit message of the first commit still claims that the first commit does it though.

(Sorry for the many iterations 😞)

Reply:
No problem! Fixed now :)